doc/content/lfr/heterogeneous_single_assembly_3D/Griffin_standalone_LFR.md

```diff
@@ -0,0 +1,277 @@
+       style=width:55%
+       id=middlefig
+       caption=Radial power density profile at the middle axial mesh of the fuel region
+
+!media lfr/lower.png
+       style=width:55%
+       id=lowerfig
+       caption=Radial power density profile at the lowest axial mesh of the fuel region
+
+!media lfr/upper.png
+       style=width:55%
+       id=upperfig
+       caption=Radial power density profile at the highest axial mesh of the fuel region
+
+[CMFDperformance] compares performance of calculations with and without CMFD using different input options using 144 processors in BEBOP high performance computing clusters of Argonne National Laboratory. The top left plot shows the total number of Richardson iterations until convergence with the relative tolerance of $10^{-4}$. The legend applies to the other three plots, too. The top right plot shows the total computing wall time (hours). The bottom left plot shows the average computing time (minutes) of transport sweeps per Richardson iteration. The bottom right plot shows the average computing time (minutes) of CMFD solves per Richardson iteration. `inner_solve_type`s of `GMRes` and `SI` with `enable_group_sweep=true` and `false`, and `inner_solve_type=none` were tested and a pure transport sweeps without the CMFD acceleration was also tested.
+
+First of all, from the top left plot, the `GMRes` inner solve type is very effective, immediately dropping the number of Richardson iterations from $39$ to $14$. Then, the number of Richardson iterations does not decrease much over increasing maximum number of inner iterations, which means the small number of inner iterations is sufficient for `GMRes`. Meanwhile, `SI` is not immediately effective, but needs larger number of inner iterations to drop the number of Richardson iterations to the similar level with `GMRes`. The number of Richardson iterations of `SI` becomes smaller than that of `GMRes` at the maximum number of inner iterations of $6$ to $7$, and thus, the total computing time, too, at $8$. The transport sweeping time of `SI` is always smaller than that of `GMRes` and turning `enable_group_sweep` on always degrades the performance because of larger transport sweeping time. The best performance is observed with `SI` without group sweeping at the maximum number of inner iterations of $9$, which shows about two times smaller computing time than the case without the CMFD acceleration. `inner_solve_type=none` with the CMFD acceleration shows about $60\%$ larger computing time than the case without the CMFD acceleration mainly because the CMFD solve time is about 10 times higher than the transport sweeping time as shown in the two bottom plots. That is, the overhead for CMFD solves is larger than the computing cost save of the reduced number of Richardson iterations. It should be noted that this performance characteristics is specific to this problem and calculation settings (fine and coarse meshes, etc.) and would be different for different problems.
```

Comment:
SI gives smaller number of Richardson iterations than GMRES with large number max inner iteration possibly because of the reflecting boundary condition, where two methods have different treatments. But this needs to have numerical proof so we do not have to include it here.

Reply:
CMFD time is pretty bad. Not sure if this is expected. Do you have the inputs? I want to run it quickly on Sawtooth to see if I can observe the same thing. Maybe @zachmprince is helping already?

Reply:
This part is updated. Now the result looks pretty good. Zach helped me. CMFD speed-up is about 10 times and this is explicitly mentioned.